testsuite/cluster-test/src/main.rs

```diff
@@ -144,41 +146,73 @@ fn setup_log() {
 
 struct ClusterUtil {
     cluster: Cluster,
-    aws: Aws,
-    prometheus: Prometheus,
+    aws: Option<Aws>,
```

Comment:
So for now we only want to support `--emit-tx` for libra swarm, everything else will require both Prometheus and Aws.
Instead of making those field optional in ClusterUtil, I think it would be cleaner to make another struct, like `BasicSwarmUtil` or something, that will have minimal number of dependencies (Cluster and Option<Prometheus>) and move emit_tx function there.

For everything else we can keep ClusterUtil as is and avoid unwrapping in all the methods of it that do require aws.

As you can see we similarly have `ClusterUtil` and `ClusterTestRunner`. `ClusterUtil` has subset of fields of `ClusterTestRunner` and it contains commands that do not require many things that `ClusterTestRunner`.

So basically we going to have
`ClusterTestRunner` - to run comprehensive tests, which contains things like HealthCheckRunner and Deployment.
`ClusterUtil` - to run simple commands on Aws cluster, prometheus is required (not an `Option<>`).
`BasicSwarmUtil` - to run commands on arbitrary bunch of nodes, does not depend on Aws at all, and have optional Prometheus. For now will have `--emit-tx` only.

Reply:
Created `BasicSwarmUtil` and created an `emit_tx` there. It is similar to ClusterUtil::emit_tx with the difference that we don't call `run_stat_loop` at the end of BasicSwarmUtil::emit_tx.

I have not added Prometheus to `BasicSwarmUtil` for now  since we don't have Prometheus running with libra-swarm. Once we do, we can add that field.